src/pybind/mgr/dashboard/frontend/src/app/shared/services/refresh-interval.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RefreshIntervalService {
+  private interval: number;
+  interval$: BehaviorSubject<number>;
```

Comment:
If we use this variable name, then you should add some documentation, otherwise i fear that a developer does not understand the intention of this variable.

If i look into Rxjs then the ``AbstractControl`` class has a variable called ``valueChanges`` or ``statusChanges`` where the name indicates the use. So i suggest to use something like ``intervalChanges``. I also suggest not to use ``$`` in the var name, though it is used for naming observables. I did not see such names of variables (which are exposed for external use like we are doing here) in Angular related frameworks (but i can be wrong here).